lib/components/narrative/default/default-itinerary.js

```diff
@@ -131,14 +131,18 @@ const ITINERARY_ATTRIBUTES = [
     id: 'cost',
     order: 2,
     render: (itinerary, options, defaultFareKey = 'regular') => {
+      debugger
```

Comment:
Remove this statement.

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/565/commits/d9c5b8309c89e75580bfaac4e812dbd10f4bda29